time_stepping/input2d.timestepping

```diff
@@ -1,22 +1,38 @@
-USE_PRECOND = TRUE
-w = 0.75
-C = 0.0
-D = 1.0
-IS_VEL_NULLSPACE = FALSE
+USE_PRECONDITIONER =  TRUE
+W = 0.75  // under relaxation factor for box relaxation scheme
+RHO = 1.0
+N = 16
+MAX_MULTIGRID_LEVELS = 3
+START_TIME = 0.0
+END_TIME = 1.0
+NUM_CYCLES = 1
+DX = 1.0 / N 
+U_MAX = 1.0 
+CFL = 0.3
+DT_MAX = CFL * DX / U_MAX 
+ENABLE_LOGGING = TRUE
+DIR_NAME = "viz2d"
+VIZ_DUMP_TIME_INTERVAL = 0.02
+VISCOUS_TIME_STEPPING_TYPE = "TRAPEZOIDAL_RULE"
+
+ETA_S = 1.0
+ETA_N = 1.0
+NU = 1.0
+XI = 1.0
 
 un {
-   function_0 = "0.0"
-   function_1 = "0.0"
+   function_0 = "cos(2*PI*X_0)*cos(2*PI*X_1)"
+   function_1 = "cos(2*PI*X_1)*cos(2*PI*X_0)"
```

Comment:
This input file is to test whether we can achieve the steady state. Setting the initial conditions to the exact solution means this problem does nothing.

We can't check error norms on steady state problems with the way the executable is set up, but I think that's ok for this problem.

Reply:
Oh, that's true. I changed it back.